### PR TITLE
Fix factory accept links

### PR DESCRIPTION
**Pull request by mkuznyetsov**

### CODE REVIEW

File: src/components/api/codenvy-factory.factory.js [hunk 1]
```diff
@@ -152,7 +152,10 @@ class CodenvyFactory {
 
       let seeLink = [];
       this.lodash.find(tmpFactory.links, function (link) {
-        if (link.rel === 'create-workspace') {
+        if (link.rel === 'accept') {
+          seeLink.push(link.href);
+        }
```
Comment (benoitf) [inline src/components/api/codenvy-factory.factory.js]: strange to use two if blocks
WDYT about
if ('accept' === link.rel || 'accept-named' === link.rel) {

Comment (mkuznyetsov) [inline src/components/api/codenvy-factory.factory.js]: right

**Comment by benoitf** [comment]
ok

**Comment by ashumilova** [comment]
+1

**Pull request merged**